test/new-e2e/tests/agent-subcommands/diagnose/diagnose_common_test.go

```diff
@@ -20,6 +21,20 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// counters contains the count of the diagnosis results
+type counters struct {
+	Total         int `json:"total,omitempty"`
+	Success       int `json:"success,omitempty"`
+	Fail          int `json:"fail,omitempty"`
+	Warnings      int `json:"warnings,omitempty"`
+	UnexpectedErr int `json:"unexpected_error,omitempty"`
+}
+
+// DiagnoseResult contains the results of the diagnose command
+type diagnoseResult struct {
+	Summary counters `json:"summary"`
+}
```

Comment:
Why are we not using the struct from the diagnose package? Is it to avoid adding to many dependencies to the test?

Reply:
importing diagnose package make the whole datadog-agent import in the go mod, I think it's a bit overkill for just a summary from my pov

Reply:
Fair point. Just to be clear I believe that `test` packages are not included as part of the final build